rmk/src/config/mod.rs

```diff
@@ -61,6 +61,9 @@ pub struct MorseConfig {
     pub mode: MorseKeyMode,
     /// If the previous key is on the same "hand", the current key will be determined as a tap
     pub unilateral_tap: bool,
+
+    // for advanced morse:
+    pub action_sets: Vec<Morse<TAP_DANCE_MAX_TAP>, 8>, //TODO MORSE_MAX_NUM = 8
```

Comment:
Now we can rename `TAP_DANCE_MAX_TAP` to `Morse_MAX_TAP`?

Reply:
This constant will disappear soon anyway, since the morse max tap will be limited in 15 only by the u16 representation.
However there will be a MORSE_MAX_NUM and a MAX_MORSE_PATTERNS_PER_KEY:

```
pub struct MorseConfig {
    pub enable_hrm: bool,
    pub prior_idle_time: Duration,
    /// Default timeout time for tap or hold
    pub operation_timeout: Duration,
    /// Default mode
    pub mode: MorseKeyMode,
    /// If the previous key is on the same "hand", the current key will be determined as a tap
    pub unilateral_tap: bool,

    // for advanced morse:
    pub action_sets: Vec<Morse<MAX_MORSE_PATTERNS_PER_KEY>, MORSE_MAX_NUM>,
}
```